scripts/kata-arch.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
```

Comment:
Just a note on the commit message - 'golang' format could also be called 'distro' or 'package' format as well - which is actually where we see a number of the issues I think - consider adding those to the description maybe.

Reply:
Fixed.

Reply:
Err - not fixed. I was thinking this too, but I'm not really sure how to make this 100% clear since, for example, `amd64` is used for Ubuntu packaging, but not for Fedora. That's why I stuck with golang for now as that's consistent but if anyone has an ISO standard / RFC spec for these naming schemes, please shout! ;)

Reply:
heh - np :-)